src/types.ts

```diff
@@ -5,7 +5,7 @@ import { AzureMonitorOpenTelemetryOptions as DistroOptions } from "@azure/monito
 import { OTLPExporterNodeConfigBase } from "@opentelemetry/otlp-exporter-base";
 
 
-export const AZURE_MONITOR_OPENTELEMETRY_VERSION = "1.0.0-beta.3";
+export const AZURE_MONITOR_OPENTELEMETRY_VERSION = "1.1.0";
```

Comment:
This should be 3.0.0-beta.10

Reply:
Oh, gotcha. Got this version mixed up with the distro version.